app/views/shared/_cohort.haml

```diff
@@ -0,0 +1,6 @@
+%li= link_to "All", root_path
+- cohorts.each do |cohort|
+  - if cohort.upcase == ENV['CLASS_ONE']
```

Comment:
Ah okay, I think I wasn't getting that we are matching the style very specifically.

This seems fine then, although we could DRY up this section and the one in `_students.html.haml` by creating a view helper to convert a cohort string to the appropriate CSS class. I can submit a PR for that later, if I feel like I have the time. 😄